src/test/java/com/reminiscence/reminiscence/journal/testEntryGetters.java

```diff
@@ -0,0 +1,116 @@
+package com.reminiscence.reminiscence.journal;
+
+import com.reminiscence.reminiscence.account.AccountRepo;
+import com.reminiscence.reminiscence.account.UserAccount;
+import com.reminiscence.reminiscence.journal.Entry;
+import com.reminiscence.reminiscence.journal.EntryRepo;
+import com.reminiscence.reminiscence.watson.ToneRepo;
+import com.reminiscence.reminiscence.watson.WatsonController;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.test.web.servlet.setup.SecurityMockMvcConfigurers;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+
+import static org.junit.Assert.*;
+
+@RunWith(SpringRunner.class)
```

Comment:
Since these are just 'normal' tests, you don't need to load up Spring.

You only need to load spring if you want to autowire something in.

Reply:
I would have tested the getter/setters in a boring class with no repos, then another test class for Repo/integration testing.